README.md

```diff
@@ -194,3 +194,44 @@ That's it! You should now be ready to cross compile your Raspberry Pi
 projects.
 
 Have fun!
+
+
+# Optional: Extending the toolset to support more system dependencies
+
+Let's say your project uses some crate that depends on having openssl
+installed on the system. In this case you have to install the package
+manually into the arm toolset.
+
+Get these packages either from the raspberry, or download them online.
+
+We'll assume that you're running Raspbian (i.e. deb files), but it should be
+straight forward to adapt the steps to your dist/packages. 32 bit is also
+assumed from here on.
+
+If you do `apt-cache show libssl1.0.0` on the raspberry, you'll see this in the
+output:
+
+    Filename:    pool/main/o/openssl/libssl1.0.0_1.0.1e-2+rvt+deb7u17_armhf.deb
+
+You should be able to find a match for that under:
+
+http://ftp.debian.org/debian/pool/
+
+If not, see if it is still on the raspberry under `/var/cache/apt/archive`.
+
+If not, try searching for the filename online.
+
+When you have the dependencies downloaded, extract them into the arm toolchain:
+
+    # cd into the sysroot of the arm toolchain
+    cd /home/vagrant/pi-tools/arm-bcm2708/arm-bcm2708hardfp-linux-gnueabi/arm-bcm2708hardfp-linux-gnueabi/sysroot/
```

Comment:
Can you replace `/home/vagrant` with `~` to make it more consistent with the rest of the guide?